### PR TITLE
[BCN-FT-102020] Jaime Pinto

### PR DESCRIPTION
**Pull request by JaimePintoP**
Dear TAs,

Here is my NPM Clone Challenge, 

- I didn't have the bold font for Poppins so everything is in regular weight. 

- I don't know if the website was supposed to be responsive, but I don't know how to do it. 

Thanks and I will be expecting for comments.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,113 @@
...
-modern open source code. See plans Join for free Bring the best of open source to your company npm is the tool used by
-over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it.
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<!DOCTYPE html>
+    <head>
+        <title>npm | build amazing things</title>
```
Comment (MartaJank) [inline index.html:3]: Same title of the page as the original, very good job! :)

File: style.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+}
+
+a {
+    text-decoration: none;
+    color:black;
+}
+
+/* HEADER */
+
+header > div {
```
Comment (MartaJank) [inline style.css:14]: Very good on using this selectors! :)

File: style.css [hunk 3]
```diff
@@ -0,0 +1,187 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
```
Comment (MartaJank) [inline style.css:2]: The way you import the font is correct but you can also choose the different font styles and this way apply the weights you want to the font.

You select your styles by clicking on "Select this style":
Untitled

Once you have the ones you like, you can embed them:
Untitled copy

This is another way of embedding the font (although you can do this as you did but with the different styles selected).
Untitled copy 2

Done! :)

Comment (JaimePintoP) [reply style.css:2]: Thanks Marta! I'm doing again the challenge with all the suggestions you gave me :)

**Pull request closed without merging**